### ISSUE TITLE
Problem when using Dynamic Services ()

### ISSUE THREAD

**Issue by nuds-zhao** [Reading-eScience-Centre/ncwms#80]
Hi,
I deployed a developer version of the ncWMS server to solve the problem in Problem when using Dynamic Services #76. But one day all layers using Dynamic Services are failed to load, with the error of "The layer *** was not found on this server". I wonder if there is any problem with this version of ncWMS.

Btw, as I observed in the past month, the problem in Issue #76 is still there, although it's less frequent to appear.

**Comment by nuds-zhao** [comment]
I find the reason why all layers can't be loaded. 
I accidentally leave one empty Alias in the server:
![empty_alias](https://user-images.githubusercontent.com/38685062/106934350-05c25600-66e8-11eb-8fe6-caf72ad27b9e.png)

It's quite difficult to identify this empty one from the default blank one. Hence I suggest that adding a condition before we match Alias with LayerName to prevent this situation in the NcwmsCatalogue.java:
![before](https://user-images.githubusercontent.com/38685062/106935321-35259280-66e9-11eb-87fc-b766c5e2c97c.png)
![after](https://user-images.githubusercontent.com/38685062/106935325-36ef5600-66e9-11eb-9d15-bd8fe0b71112.png)

**Comment by guygriffiths** [comment]
I'm not sure I follow.  `layerName.startsWith(null)` will always return false - so explicitly testing that `testDynamicService.getAlias() != null` will never make a difference to that condition.

**Comment by nuds-zhao** [comment]
Yeah, you are right. I made a mistake here. The empty Alias should correspond to an empty string. `null`  may represent the default condition, so it would not make any difference. How about `layerName.startsWith('')`? What would that return? I think the correct condition to test should be` testDynamicService.getAlias() != ''`.
